.github/ISSUE_TEMPLATE/4-new-hint-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F4A1Hint request"
+about: Suggest a new hint for webhint
+
+---
+<!--
+
+Hi there 👋
+
+Before opening a new issue please make sure to search in the existing
```

Comment:
Another positive, thankful opening would be nice (similar to those above). _Especially_ for new hints we should be thankful as we'll benefit a lot from crowdsourcing the web for these (as we already have).